schema/tests/Kam/TrunksCdr/TrunksCdrRepositoryTest.php

```diff
@@ -22,7 +22,8 @@ public function test_runner()
         $this->it_finds_one_by_callid();
         $this->it_finds_unparsedCalls();
         $this->it_resets_parsed_calls();
-        $this->it_resets_orphan_cgrids();
+        $this->it_get_cgrids_by_billableCalls();
+//        $this->it_resets_orphan_cgrids();
```

Comment:
Is this right?

Reply:
Nope, good catch, this was done while testing each of the functions individually because I was not sure what the source of the error was :smile: